src/styles/base/_base.scss

```diff
@@ -0,0 +1,22 @@
+html {
+    font-family: sans-serif;
+    font-size: 62.5%;
```

Comment:
is it really a good number for font-size?? for the most part font-size is defined in pixels

Reply:
Fixed.